Compilation fails after adding Dock Manager component template

### Description  
Compilation fails after adding Dock Manager component template.

- Ignite UI CLI version: 6.1.0
- Framework: Angular
- Project type: igx-ts

### Steps to reproduce  

1. Create new Angular project.
2. Add Dock Manager component template with a custom component name.
3. Run the project.

### Result  
Compilation fails:
ERROR in src/app/app.module.ts:27:5 - error TS2304: Cannot find name 'DockManagerModule'.

### Expected result  
Compilation should be successful.

--- a/packages/igx-templates/igx-ts/dock-manager/default/index.ts
+++ b/packages/igx-templates/igx-ts/dock-manager/default/index.ts
@@ -13,7 +13,7 @@ class IgcDockManagerTemplate extends IgniteUIForAngularTemplate {
 		this.name = "Dock Manager";
 		this.description = "basic IgcDockManager";
 		this.dependencies = [
-			{ import: "DockManagerModule", from: "./src/app/<%=path%>/<%=filePrefix%>.module.ts" }
+			{ import: "<%=ClassName%>Module", from: "./src/app/<%=path%>/<%=filePrefix%>.module.ts" }
 		];
 		// "igniteui-dockmanager@~1.0.0":
 		this.packages = [ `${resolveIgxPackage(NPM_DOCK_MANAGER)}@~1.0.0"` ];

